package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "dotenv": "^16.0.3",
```

Comment:
💯 but you actually don't need this.

Check out this documentation for react env vars.
[react-env-vars](https://create-react-app.dev/docs/adding-custom-environment-variables/)

Try to change your env vars so you don't need to install `dotenv` manually